etc/config-template.toml

```diff
@@ -344,7 +344,19 @@
 ## Compaction and flush can cause terrible spikes if they exceed a certain threshold. Consider
 ## setting this to 50% ~ 80% of the disk throughput for a more stable result. However, in heavy
 ## write workload, limiting compaction and flush speed can cause write stalls too.
+## 1. rate-bytes-per-sec is the only parameter you want to set most of the time. It controls the
+##    total write rate of compaction and flush in bytes per second. Currently, RocksDB does not
+##    enforce rate limit for anything other than flush and compaction, e.g. write to WAL.
+## 2. rate-limiter-mod indicates which types of operations count against the limit.
+##    1 : ReadOnly
+##    2 : WriteOnly
+##    3 : AllIo
+## 3. auto_tuned enables dynamic adjustment of rate limit within the range
+##    [rate_bytes_per_sec / 20, rate_bytes_per_sec], according to the recent demand for background I/O.
 # rate-bytes-per-sec = 0
+# rate-limiter-mod = 2
```

Comment:
@yiwu-arbug 

Is it better to use 3 instead of 2 in default?
Which one did you use before in production?

Reply:
I use 2 in production. And when not use "auto-tuned", it is mod 2 in default. So I think maybe it is better to use mod 2 in default.

Reply:
Using kWritesOnly is normal. kAllIo is used only when read+write rate is so high that IO bandwidth is saturated and we want to avoid starving reads.

Reply:
Are we able to use string ("read-only", "write-only" and "all-io") instead of int to make the config more human readable?

Reply:
s/mod/mode/?

Reply:
> Are we able to use string ("read-only", "write-only" and "all-io") instead of int to make the config more human readable?

Here, like other config "wal-recovery-mode" and "compaction-pri", those all use int. If needed, I may file another issue to change all configs like this.

Reply:
> s/mod/mode/?

I will fix it.

Reply:
> @fredchenbj looks good overall. Is it possible to change the numeric "rate-limiter-mode" to a string type? For example, we have the compression type which is also enum on rocksdb side, but being readable strings in the tikv config: https://github.com/tikv/tikv/blob/master/src/util/config.rs#L84

https://github.com/tikv/tikv/pull/4335#discussion_r264490104 . Yes, we can change it to string. Maybe I could do it in another pr, and change some other configs to string too.